res/online-maps/mapconfig.json

```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "name": "OpenStreetMap",
+        "servers": [
+            ".tile.openstreetmap.org"
+        ],
+        "copyright": "Map tiles (c) OpenStreetMap (ODbL)",
+        "url": "{z}/{x}/{y}.png",
+        "min_zoom_level": 1,
+        "max_zoom_level": 17
+    },
+    {
+        "name": "OpenTopoMap",
+        "servers": [
+            ".tile.opentopomap.org"
+        ],
+        "copyright": "Map Data (c) OpenStreetMap, SRTM - Map Style (c) OpenTopoMap (CC-BY-SA)",
+        "url": "{z}/{x}/{y}.png",
+        "enabled": false
```

Comment:
I agree that OpenTopo should be moved here, your new implementation can replace all existing online tile sources. While OpenTopo should be enabled by default at that point, I think that OpenStreetMap should better be left disabled in the default config. The reason why I chose OpenTopo instead of OpenStreetMap is that its usage policy actually allows 3rd party apps to access their servers while I believe that OpenStreetMap explicitly disallows that ("Our data is free, but the tile servers are not"). I last checked that in 2018 or so, maybe it changed. I'll check.